src/Raven.Server/SqlMigration/SqlEmbeddedTable.cs

```diff
@@ -54,7 +56,13 @@ public SqlReader GetReaderWhere(List<string> values)
             }
 
             Reader = new SqlReader(Connection, query, true);
+            Reader.ExecuteReader();
             return Reader;
         }
+
+        public static explicit operator SqlEmbeddedTable(List<string> v)
```

Comment:
Avoid any such usage. `operator` should only be used very rarely and not for things like this